wattile/entry_point.py

```diff
@@ -35,6 +35,7 @@ def init_logging(local_results_dir):
     logger.addHandler(hdlr)
     logger.setLevel(logging.INFO)
     logger.info("PID: {}".format(PID))
+    logger.info("Trained with Wattile version: {}".format(wattile_version))
```

Comment:
Can we also get this somewhere in a configuration file output? Or is it stored as a property of the model object? SkySpark may need to be able to read the version out of the trained model config.

Reply:
Absolutely. Adding the version to the output.out file is a good conversation starter, but we can absolutely add this somewhere that is easier to parse by machines. 

The generated model file is a torch model, so doesn't feel like the right spot to put the metadata. I don't see existing artifacts that seem like an obvious place to add this (screenshot below of example model artifacts), but happy for feedback as I could be missing something obvious. 

<img width="295" alt="Screenshot 2024-07-23 at 11 03 52 AM" src="https://github.com/user-attachments/assets/586b5319-b4f0-47d6-b39c-390b9bbf452a">


I'm wondering if we should we add a separate mode_metadata file. It could minimally include the `wattile_version`, but perhaps we'll want to add more in the future - like specifics about the training ENV (some of which are captured by output.out). We could potentially even replace the output.out file with a machine readable metadata file. 

What do y'all think?